src/types.ts

```diff
@@ -62,3 +71,33 @@ export type ChainID = number;
 export enum ColonyActionType {
   MintTokens = 'MINT_TOKENS',
 }
+
+export enum ColonyMotionType {
+  MintTokensMotion = 'MINT_TOKENS_MOTION',
+}
+/*
+ * Contract calls
+ */
+export enum ColonyActions {
```

Comment:
Could the name be more indicative that these are the contract calls?

Reply:
I've gone for `ColonyOperations`. Preferable?